src/components/CommandBar/CommandBarHeader.tsx

```diff
@@ -161,7 +168,9 @@ function CommandBarHeader({ children }: React.PropsWithChildren<object>) {
                             ),
                             4
                           )
-                        ) : arg.inputType === 'text' && !arg.valueSummary ? (
+                        ) : arg.inputType === 'text' &&
+                          !arg.valueSummary &&
+                          typeof argValue === 'string' ? (
                           `${argValue.slice(0, 12)}${argValue.length > 12 ? '...' : ''}`
```

Comment:
Good catch, off-topic but this whole ternary is so dense at some point we should break it up